File: pause/pause.go
package main

import "gocv.io/x/gocv"

func main() {
	// video, err := gocv.VideoCaptureFile("../video/video.mp4")
	// if err != nil {
	// 	// return nil, err
	// 	panic(err)
	// 	// return err
	// }
	webcam, _ := gocv.VideoCaptureDevice(0)
	window := gocv.NewWindow("Hello")
	img := gocv.NewMat()

	for {
		webcam.Read(&img)
		window.IMShow(img)
		window.WaitKey(1)
	}

	// return nil
}

File: pause/pause.py
import cv2

cap = cv2.VideoCapture('../video/video.mp4')
if not cap.isOpened():
    print("Error opening video")

while(cap.isOpened()):
    status, frame = cap.read()
    if status:
        cv2.imshow('frame', frame)
    key = cv2.waitKey(0)

    # if key == 1:
    #     cv2.waitKey()
    # elif key == ord('q'):
    #     break